custom_components/myskoda/device_tracker.py

```diff
@@ -57,7 +57,10 @@ def latitude(self) -> float | None:  # noqa: D102
 
         self._update_device_from_coordinator()
 
-        return self.vehicle.position.positions[0].gps_coordinates.latitude
+        if len(self.vehicle.position.positions) > 0:
+            return self.vehicle.position.positions[0].gps_coordinates.latitude
+        else:
+            return None
```

Comment:
nit: if we enable all ruff rules (not done here yet) it will trip over https://docs.astral.sh/ruff/rules/superfluous-else-return/

```suggestion
        return None
```